From a86424dcfd22d1bd057db714f7fdef2ecf1c5a31 Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@freescale.com>
Date: Fri, 28 Dec 2012 12:25:47 -0200
Subject: [PATCH 1/9] ARM: mach-imx: Kconfig: Do not select Babbage for
 MACH_IMX51_DT

When mx51 dt support was added it was necessary to select MACH_MX51_BABBAGE in
order to get the mx51 dt built without errors.

At that time, there was no pinctrl support for mx51 yet.

Currently there is no need to select MACH_MX51_BABBAGE anymore.

Reported-by: Matt Sealy <matt@genesi-usa.com>
Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/mach-imx/Kconfig |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-imx/Kconfig b/arch/arm/mach-imx/Kconfig
index 0a2349d..9319793 100644
--- a/arch/arm/mach-imx/Kconfig
+++ b/arch/arm/mach-imx/Kconfig
@@ -756,7 +756,6 @@ comment "i.MX51 machines:"
 
 config MACH_IMX51_DT
 	bool "Support i.MX51 platforms from device tree"
-	select MACH_MX51_BABBAGE
 	select SOC_IMX51
 	help
 	  Include support for Freescale i.MX51 based platforms
-- 
1.7.10.4

